frontend/src/components/NewPassword/NewPassword.js

```diff
@@ -0,0 +1,105 @@
+/* global sessionStorage */
+/* eslint no-undef: "error" */
+
+import React, { Component } from "react";
+import { loginUser } from "../../lib/apiClient";
+import "./NewPassword.css";
+
+class SignIn extends Component {
```

Comment:
I believe this should match the file name of the component and describe what it's doing (I think this component was just copied from SignIn, but would make sense to rename it to NewPassword to be consisten with this new component).